backend/src/Equinor.ProjectExecutionPortal.WebApi/appsettings.json

```diff
@@ -25,12 +25,12 @@
     "Environment":  "ci"
   },
   "AzureAd": {
-    "ClientId": "", // This app's AppReg key
+    "ClientId": "guid", // This app's AppReg key
     "TenantId": "3aa4a235-b6e2-48d5-9195-7fcf05b459b0",
     "Instance": "https://login.microsoftonline.com",
     "CallbackPath": "/signin-oidc",
     "SignedOutCallbackPath ": "/signout-callback-oidc",
-    "ClientSecret": ""
+    "ClientSecret": "hushhush"
```

Comment:
det skal være hushhush?

Reply:
For at testene skal kjøre må feltet være populert. I praksis benyttes det ikke, og verdien kan være hva som helst